templates/systemd.service.j2

```diff
@@ -12,6 +12,8 @@ EnvironmentFile={{ vaultwarden_directory }}/.env
 User = vaultwarden
 Group = vaultwarden
 ExecStart = {{ vaultwarden_directory }}/vaultwarden
+Restart=on-failure
+RestartSec=5s
```

Comment:
How about going for `After=database.service` instead?

Reply:
the problem is we're not sure what database system the user is using, or if there even is one (E.g. sqlite)

Reply:
Systemd will ignore services that don't exist. Worst case is that the service will wait for a database it doesn't use.

Reply:
I might even try to change this on the vaultwarden side, since diesel is having trouble, but the migrations start running later, when the db has started...

Reply:
Ideally databases run with `Type=notify`, so they can tell systemd via sd_notify, when they're up. I know postgres supports this, and I'm currently not using any other DBMS actively.